homeassistant/components/reolink/__init__.py

```diff
@@ -108,10 +109,21 @@ async def async_setup_entry(
                 config_entry.data[CONF_PORT],
                 host.api.port,
             )
+        if (
+            config_entry.data.get(CONF_BC_PORT, host.api.baichuan.port)
+            != host.api.baichuan.port
+        ):
+            _LOGGER.warning(
```

Comment:
Why do we need to warn the user here?

Reply:
Because the port should not be changing. If it does there is some underlying problem (probably in my code or the device is sending bogus data), which I want to know about.

Unless the user changes the port manually in the reolink desktop app, this warning will never be shown. If the user did change it, it will only be shown 1x.